snapshot/lib/snapshot/options.rb

```diff
@@ -122,7 +122,7 @@ def self.available_options
                                      optional: true,
                                      description: "The bundle identifier of the app to uninstall (only needed when enabling reinstall_app)",
                                      code_gen_sensitive: true,
-                                     default_value: ENV["SNAPSHOT_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
+                                     default_value: ENV["SNAPSHOT_APP_IDENTIFIER"] || ENV["SNAPSHOT_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
```

Comment:
Same here. `SNAPSHOT_APP_IDENTIFIER` is already set as the `env_name`.  Is this just duplicate code? 🤔